# std_msgs/Header header
# uint64 count
int64 time_stamp
uint32 id
uint8 rtr
uint8 ext
int32 data_len
uint8[] data

# CANFD
